CMakeLists.txt

```diff
@@ -646,7 +646,7 @@ else() #!WIN32
         set(MARCH -march=native)
     endif()
 
-    set(CMAKE_C_FLAGS_DEBUG "-Og -fno-omit-frame-pointer")
+    set(CMAKE_C_FLAGS_DEBUG "-O0 -fno-omit-frame-pointer")
```

Comment:
JFYI. You could also selectively disable some optimizations, e.g. `-fno-inline-functions`.

Reply:
Yes, but I am not sure it is worth hunting for them. Unless -O0 becomes a performance problem even in debug mode, the optimization with -Og were making debugging completely impossible for me.

(I may not have the same definition of "greatly interfere with debugging" than GCC people).

Reply:
> (I may not have the same definition of "greatly interfere with debugging" than GCC people).

:wink: